www/js/control/general-settings.js

```diff
@@ -19,7 +21,9 @@ angular.module('emission.main.control',['emission.services',
                                         'emission.survey.enketo.demographics',
                                         'emission.plugin.logger',
                                         'emission.config.dynamic',
-                                        QrCode.module])
+                                        QrCode.module,
+                                        ControlDataTable.module,
+                                        ControlListItem.module])
```

Comment:
For my own edification: do we need to import `ControlListItem` here?
I don't see a `ControlListItem.jsx`; is it missing?

Reply:
You're right, that's unnecessary. It was a part of something else, I had started to tinker with changing a different piece of that page